### DISCUSSION TITLE
Which monorepo type to choose for an Angular Nestjs app?

### DISCUSSION THREAD

**Discussion by gabrielnemeth** [nrwl/nx#15705] [General]
Hey there,

I want to create an Angular+Nestjs combo app. Previously (in older nx version) there was an option to generate such monorepo. I'm a bit confused with the new options. From what I read already the best way is to generate an integrated repo and add both Angular and Nestjs through plugins. Or is there another “better” way?

**Comment by AgentEnder** [comment]
Nope, you've got it correct. Pick integrated -> Angular or Apps preset and generate what you need.

**Comment by gajanannarwade1412** [comment]
Try this hope it will helpfull to you:
https://github.com/gajanannarwade1412/nest-angular-monorepo
